moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -142,6 +144,16 @@ MotionPlanningDisplay::MotionPlanningDisplay()
                               plan_category_, SLOT(changedQueryMarkerScale()), this);
   query_marker_scale_property_->setMin(0.0f);
 
+  query_marker_offset_category_ = new rviz::Property("Interactive Marker Offset", QVariant(), "", plan_category_, nullptr, this);
```

Comment:
Please provide more description details here. W.r.t. which coordinate system is the offset described?